package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vscode-chrome-debug-core",
   "displayName": "vscode-chrome-debug-core",
-  "version": "6.7.21",
+  "version": "6.7.22",
```

Comment:
Don't change this version, I'll do it when I publish it (which I can do right now but this is easier to manage if this version isn't changed in a PR)

Reply:
Oh, ok! Sorry about that, undoing it now.